scc/project-run-anyuid-template.yml

```diff
@@ -20,7 +20,7 @@ objects:
   fsGroup:
     type: RunAsAny
   apiVersion: v1
-  priority: "${PRIORITY_LEVEL}"
+  priority: "${{PRIORITY_LEVEL}}"
```

Comment:
Are you sure you want to wrap a non-string parameter in quotes?

Reply:
We could change that if you think it makes more sense but one the template is processed it doesn't make any difference as the value appears as an 'int':

```
mak 🎩︎ @ MakF ~/GIT/openshift-templates/scc (fix/scc-template)
└─ ▶ grep priority: project-run-anyuid-template.yml
  priority: "${{PRIORITY_LEVEL}}"

mak 🎩︎ @ MakF ~/GIT/openshift-templates/scc (fix/scc-template)
└─ ▶ oc process -f project-run-anyuid-template.yml NAME=dummy NAMESPACE=dummy|grep priority
            "priority": 8,
```

Reply:
It works either way but my argument would be to remove the quotes just to clarify/emphasize that this is an integer. I can go either way, just let me know @makentenza and I will approve and merge.

Reply:
@pabrahamsson @etsauer it's fine by me, just removed quotes for this parameter.